sdcm/nemesis.py

```diff
@@ -3687,6 +3706,14 @@ def disrupt(self):
         self.disrupt_rolling_restart_cluster(random=False)
 
 
+class ToggleInternodeComm(Nemesis):
```

Comment:
Add a new property to this nemesis (later we can add to other similar ones) mentioning this nemesis is doing "Full Cluster Restart"

Reply:
done